github_plugin/transformer_functions.py

```diff
@@ -0,0 +1,191 @@
+import base64
+
+from external_plugins.github_integrate import default as DEFAULT 
```

Comment:
change the github_integrate  to github_plugin

Reply:
done